game.js

```diff
@@ -0,0 +1,40 @@
+class Game {
+
+  constructor() {
+    this.rolls = [];
+  }
+  
+  
+  roll(pins){
+    this.rolls.push(pins);
+  }
+
+  get points() {
+    let points = 0;
+    let rollIndex = 0;
+
+    // calculates points per game frames(therefore 10 is the iteration value)
+    for (let frame = 0; frame < 10; frame++) {
+      
+      // isStrike?
+      if(this.rolls[rollIndex] === 10) {
+        points += 10 + this.rolls[rollIndex + 1] + this.rolls[rollIndex + 2];
+        rollIndex ++;
+        continue;
+      } 
+
+      const framePoints = this.rolls[rollIndex] + this.rolls[rollIndex + 1];  
```

Comment:
I'm not sure if this works for the last frame. If a strike was rolled on the first shot of frame 10, we then shouldn't be adding the 2nd and 3rd shots for example (they are only added as bonus). Essentially I'm unsure if your logic fully deals with frame 10 (I might be wrong though)